fulp_modules/features/sect/earth_cult.dm

```diff
@@ -0,0 +1,154 @@
+/datum/religion_sect/earth
+	name = "Earthen God"
+	quote = "From the earth you came; to the earth you shall return."
+	desc = "The Earth is alive and loves her children. Even here you can feel her heartbeat under your feet."
+	tgui_icon = "robot"
+	alignment = ALIGNMENT_NEUT
+	desired_items = list(/obj/item/reagent_containers = "holding blood", /mob/living/basic/bull)
+	rites_list = list(/datum/religion_rites/pure_heart, /datum/religion_rites/big_cow, /datum/religion_rites/bull_sacrifice)
+	altar_icon_state = "convertaltar-blue"
+	max_favor = 10000
+
+/obj/item/organ/internal/heart/cybernetic/earth
+	name = "pure heart"
+	desc = "A pure, undying heart that beats in time with the Earth's."
+	icon = 'fulp_modules/features/sect/earth-cult.dmi'
+	icon_state = "heart-on"
+	organ_flags = ORGAN_SYNTHETIC
+	maxHealth = 1.5 * STANDARD_ORGAN_THRESHOLD
+	dose_available = FALSE
+	emp_vulnerability = 0
+
+
+/obj/item/organ/internal/heart/cybernetic/earth/Insert(mob/living/carbon/heart_owner, special = FALSE, drop_if_replaced = TRUE)
+	. = ..()
+	ADD_TRAIT(heart_owner, TRAIT_SPIRITUAL, ORGAN_TRAIT)
+	ADD_TRAIT(heart_owner, TRAIT_VIRUSIMMUNE, ORGAN_TRAIT)
+	owner.faction |= "earth"
+
+
+/obj/item/organ/internal/heart/cybernetic/earth/Remove(mob/living/carbon/heart_owner, special = FALSE)
+	REMOVE_TRAIT(heart_owner, TRAIT_SPIRITUAL, ORGAN_TRAIT)
+	REMOVE_TRAIT(heart_owner, TRAIT_VIRUSIMMUNE, ORGAN_TRAIT)
+	owner.faction -= "earth"
+	return ..()
+
+/obj/item/organ/internal/heart/cybernetic/earth/on_life(mob/living/carbon/heart_owner, )
+	if(istype(get_turf(owner), /turf/open/floor/grass))
+		owner.set_timed_status_effect(2 SECONDS, /datum/status_effect/speed_boost, only_if_higher = TRUE)
+	return ..()
+
+/obj/item/organ/internal/heart/cybernetic/earth/Insert(mob/living/carbon/brain_owner, special, drop_if_replaced, no_id_transfer)
+    . = ..()
+    if(!ishuman(brain_owner))
```

Comment:
DNA is on the carbon level, I don't think this check & ``var/mob/living/carbon/human/human_receiver = brain_owner`` is necessary